plugins/yoga/src/create-yoga.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  GraphQLOTELContext,
+  GraphQLOTELContextOptions,
+  TraceSchemaInput,
+  traceSchema,
+} from "@graphql-debugger/trace-schema";
+
+import { GraphQLSchema } from "graphql";
+import {
+  YogaServerInstance,
+  YogaServerOptions,
+  createYoga as originalCreateYoga,
+} from "graphql-yoga";
+
+interface DebuggerYogaOptions<TServerContext, TUserContext>
+  extends YogaServerOptions<TServerContext, TUserContext> {
+  debugger?: Omit<TraceSchemaInput, "schema"> & {
+    shouldDisable?: boolean;
+    otelContextOptions?: GraphQLOTELContextOptions;
+    traceSchemaOptions?: Omit<TraceSchemaInput, "schema">;
```

Comment:
@akash-joshi here we pass thru the otel context options and trace schema options so users can configure 3rd party tracing and or point to a different collector.